example-next/pages/index.tsx

```diff
@@ -88,22 +84,23 @@ function Home(): NextPage {
         feePayer: 'from'
       }).then(fee => console.log({ fee }))
 
-      if (to && amountInLamports) {
+      if (to && amountInLamports)
         signAndSendTransaction('transfer', {
           to,
           amountInLamports,
           feePayer: 'from'
         }).then(async result => {
           console.log({ result })
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
In general, I stick to a rule where if you have to define eslint disable often, it is worth to just disable it in general config file, maybe thats the case here.